### ISSUE TITLE
PHP Catchable fatal error when trying to access "Special:Statistics"

### ISSUE THREAD

**Issue by kghbln** [SemanticMediaWiki/SemanticMediaWiki#1123]
I am not exactly sure when this started but it must be a fairly new issue.
```text
PHP Catchable fatal error: 
Argument 3 passed to SMW\\MediaWiki\\Hooks\\SpecialStatsAddExtra::__construct() must be an instance of Language,
instance of StubUserLang given,
called in /.../w/extensions/SemanticMediaWiki/src/MediaWiki/Hooks/HookRegistry.php on line 284 and
defined in /.../w/extensions/SemanticMediaWiki/src/MediaWiki/Hooks/SpecialStatsAddExtra.php on line 72
```
I am currently using MW 1.24.3 and SMW master (8d0aebd) on PHP 5.4.41

**Comment by mwjames** [comment]
Relates to #1118, where I thought I can have access to `$GLOBALS['wgLang']` before it is unstubbed but apparently this is not the case. Since this is hard to test it occurs only during "real" interaction.

**Comment by mwjames** [comment]
Fixed with de15e1a

**Comment by kghbln** [comment]
Wow, another fast one. Thanks a bunch! :)
